tests/cupy_tests/sorting_tests/test_sort.py

```diff
@@ -650,6 +650,22 @@ def test_argpartition_multi_dim(self, xp, dtype):
                 a[rows, cols, idx[:, :, kth + 1:]]).all()
         return idx[:, :, kth:kth + 1]
 
+    @testing.for_all_dtypes(no_bool=True)
+    @testing.numpy_cupy_array_equal(accept_error=True)
```

Comment:
For some reason, the comparison for integer types for both CuPy and NumPy fails

Reply:
let me kick CI, will see the failure.

Reply:
Do you want me to remove the `accept_error` argument?

Reply:
I got no error on my local, so maybe something wrong in your environment. Please just remove `accept_error` that we don't need.